backend/src/migrations/1673503553138-metloConfigEnv.ts

```diff
@@ -0,0 +1,76 @@
+import { AppDataSource } from "data-source"
+import jsyaml from "js-yaml"
+import { MetloConfig } from "models/metlo-config"
+import { getEntityManager } from "services/database/utils"
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { decrypt, encrypt, generate_iv } from "utils/encryption"
+
+export class metloConfigEnv1673503553138 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "metlo_config" ADD COLUMN IF NOT EXISTS "env" text`,
+    )
+    await queryRunner.query(
+      `ALTER TABLE "metlo_config" ADD COLUMN IF NOT EXISTS "envTag" text`,
+    )
+    await queryRunner.query(
+      `ALTER TABLE "metlo_config" ADD COLUMN IF NOT EXISTS "envIV" text`,
+    )
+
+    const entityManager = AppDataSource.createEntityManager(queryRunner)
+
+    const configs = await entityManager.find(MetloConfig, {
```

Comment:
we can just get rid of this part, can cause issues in the future if we ever change this